### PR TITLE
feat: add words

### PR DESCRIPTION
**Pull request by cloydlau**

### CODE REVIEW

File: dict.json [hunk 1]
```diff
@@ -67,13 +78,17 @@
   "esm": "ESM",
   "esxi": "ESXi",
   "etcd": "etcd",
+  "event loop": "Event Loop",
+  "eventloop": "EventLoop",
```
Comment (antfu) [inline dict.json:82]: This is too general to be considered as a term IMO

File: dict.json [hunk 2]
```diff
@@ -221,12 +238,14 @@
...
   "powerpoint": "PowerPoint",
   "powershell": "PowerShell",
+  "primevue": "PrimeVue",
   "protobuf": "Protobuf",
   "pwa": "PWA",
   "pycharm": "PyCharm",
   "pytorch": "PyTorch",
   "qemu": "QEMU",
   "qtscript": "QtScript",
+  "quasar": "Quasar",
```
Comment (antfu) [inline dict.json:248]: Please read the guideline

File: dict.json (hunk 1 above)
Comment (cloydlau) [reply dict.json]: removed

File: dict.json [hunk 3]
```diff
@@ -67,13 +78,17 @@
...
+  "eventloop": "EventLoop",
   "faas": "FaaS",
   "facetime": "FaceTime",
   "fastapi": "FastAPI",
   "ffmpeg": "FFmpeg",
   "filezilla": "FileZilla",
   "firefox": "Firefox",
   "freemarker": "FreeMarker",
+  "front end": "Front End",
+  "frontend": "FrontEnd",
```
[the remark this answers is not in the document]
Comment (cloydlau) [reply dict.json]: removed

File: dict.json (hunk 2 above)
Comment (cloydlau) [reply dict.json]: removed this and 'Vuetify‘, 'Axios'

File: dict.json [hunk 4]
```diff
@@ -284,6 +303,7 @@
   "tsdx": "TSdx",
   "typescript": "TypeScript",
   "udp": "UDP",
+  "ui": "UI",
```
[the remark this answers is not in the document]
Comment (cloydlau) [reply dict.json]: removed

**Pull request merged**